File: android/other/AsyncTaskTest/app/src/main/java/com/example/asynctasktest/MainActivity.java
package com.example.asynctasktest;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ProgressBar;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private TextView texttitle;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView texttitle = findViewById(R.id.txttitle);
        ProgressBar pgbar = findViewById(R.id.pgbar);
        Button btnupdate = findViewById(R.id.btnupdate);
        btnupdate.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                MyAsyncTask myTask = new MyAsyncTask(texttitle, pgbar);
                myTask.execute(1000);
            }
        });

    }
}

File: android/other/LogUtil/app/src/main/java/com/example/logutil/MainActivity.kt
package com.example.logutil

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

private const val TAG = "MainActivity"

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        LogUtil.v(TAG, "VERBOSE")
        LogUtil.d(TAG, "DEBUG")
        LogUtil.e(TAG, "ERROR")
    }
}